lib/lints/avoid_late_keyword/models/avoid_late_keyword_parameters.dart

```diff
@@ -0,0 +1,20 @@
+/// A data model class that represents the "avoid late keyword" input
+/// parameters.
+class AvoidLateKeywordParameters {
+  /// Maximum number of parameters
+  final bool allowInitialized;
```

Comment:
Copy-pasted comment is wrong.
```suggestion
  /// Maximum number of parameters
  final bool allowInitialized;
```